doc/Getting_Started/Tutorials/Quick_Start.md

```diff
@@ -17,71 +32,73 @@ and is able to play a single content at once.
 
 To instanciate it with a linked video element you can just do something along
 the lines of:
+
 ```js
 import RxPlayer from "rx-player";
 
 const videoElement = document.querySelector("video");
 const player = new RxPlayer({ videoElement });
 ```
 
-``videoElement`` is an RxPlayer option and will be the HTMLElement the RxPlayer
+`videoElement` is an RxPlayer option and will be the HTMLElement the RxPlayer
 will load your media on.
 
 Despite its name, you can also give it an `<audio>` element. It will still be
 able to play an audio content without issue.
 
 When you are ready to make use of more advanced features, you can look at the
-other possible options in the [Player Options page](../api/player_options.md).
-
-
+other possible options in the [Player Options
+page](../../api/Creating_a_Player.md).
```

Comment:
It's a small thing, I saw that behavior occurred multiple times, but it shouldn't be on a single line?

```
other possible options in the [Player Options page](../../api/Creating_a_Player.md).
```

Reply:
I do that on purpose to avoid having lines longer than 80 columns, to facilitate reading in text editors.

What's the problem with it being on multiple lines?

Reply:
I just find it harder to read but I also completely understand your point of view of not having line long than 80 columns.

So its ok for me :)